ultraplot/internals/rcsetup.py

```diff
@@ -1197,6 +1197,11 @@ def copy(self):
         _validate_bool,
         "Whether to trim trailing decimal zeros on tick labels.",
     ),
+    "formatter.log": (
+        False,
+        _validate_bool,
+        "Whether to use log scale for semilog or loglog plots.",
```

Comment:
```suggestion
        "Whether to use log formatting (e.g., $10^{4}$) for "
        "logarithmically scaled axis tick labels.",
```

This doc string is confusing. The term "scale" is used to refer to how a unit of distance on the axis translate into a change in the data. Here you mean how the axis labels are formatted.